osv/feedback.json

```diff
@@ -0,0 +1,99 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://raw.githubusercontent.com/google/osv.dev/main/osv/invalid_records_schema.json",
+    "type": "object",
+    "default": {},
+    "title": "The OSV.dev invalid record Schema",
+    "required": [
+        "invalid_records"
+    ],
+    "properties": {
+        "invalid_records": {
+            "type": "array",
+            "default": [],
+            "title": "The OSV.dev per-source invalid records",
+            "items": {
+                "type": "object",
+                "title": "An individual invalid record",
+                "required": [
+                    "id",
+                    "finding_first_seen",
+                    "import_last_attempt",
+                    "findings"
+                ],
+                "properties": {
+                    "id": {
+                        "type": "string",
+                        "title": "The id of the OSV record",
+                        "examples": [
+                            "GSD-2021-1001229",
+                            "GSD-2022-1003535"
+                        ]
+                    },
+                    "import_last_attempt": {
+                        "type": "string",
+                        "title": "When an import for this record was last attempted in RFC3339",
```

Comment:
nit: Should we use ISO8601 instead?, it seems to be the more popular format from what I understand. (I think they are both very similar)

e.g. python has a toisoformat and fromisoformat which uses 8601. but I know golang only has RFC3339.

Apparently neither is a superset of the other...  https://ijmacd.github.io/rfc3339-iso8601/

Reply:
Yes, RFC3339 is indeed an annoying variant. https://xkcd.com/927/
I went with RFC3339 for consistency with the rest of the OSV Schema.